language/stdlib/modules/DualAttestation.move

```diff
@@ -130,8 +130,11 @@ module DualAttestation {
     }
     spec fun rotate_base_url {
         let sender = Signer::spec_address_of(account);
-        include LibraTimestamp::AbortsIfNoTime;
+
+        /// Must abort if the account does not have the resource Credential [B25].
         include AbortsIfNoCredential{addr: sender};
+
+        include LibraTimestamp::AbortsIfNoTime;
         ensures global<Credential>(sender).base_url == new_url;
 
         /// The sender can only rotates its own base url [B25].
```

Comment:
rotates --> rotate (let's fix typos in comments when convenient).

Reply:
Done.